go-controller/pkg/metrics/metrics.go

```diff
@@ -156,6 +158,30 @@ func getCoverageShowOutputMap(component string) (map[string]string, error) {
 	return coverageShowMetricsMap, nil
 }
 
+// ovnKubeLogFileSizeMetricsUpdater updates the metrics that obtains the
+// size of ovnkube process' logfile
+func ovnKubeLogFileSizeMetricsUpdater(ovnKubeLogFileMetric *prometheus.GaugeVec,
+	stopChan <-chan struct{}) {
+	ticker := time.NewTicker(metricsUpdateInterval)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			logfile := config.Logging.File
```

Comment:
`logFile`

Reply:
logfile is a common usage elsewhere in kubernetes code, so going to keep what I have.